src/main.test.js

```diff
@@ -2,7 +2,7 @@ const envSpecToHTML = require("./main.js");
 
 test("Valid input with type", () => {
   const testEnv = "DATABASE_URL\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpecToHTML(testEnv)).resolves.toEqual(
```

Comment:
We better test `envSpecToHTML.parse` here instead.

Also it would be a good idea to rename `envSpecToHTML` to `envSpec` in the first line — `envSpec.parse` makes more sense as a name 😄.